src/nix/flake.md

```diff
@@ -153,18 +161,39 @@ can occur in *locked* flake references and are available to Nix code:
 
 Currently the `type` attribute can be one of the following:
 
-* `path`: arbitrary local directories, or local Git trees. The
-  required attribute `path` specifies the path of the flake. The URL
-  form is
+* `indirect`: *The default*. Indirection through the flake registry.
+  These have the form
+
+  ```
+  [flake:]<flake-id>(/<rev-or-ref>(/rev)?)?
+  ```
+
+  These perform a lookup of `<flake-id>` in the flake registry. For
```

Comment:
FYI: Following recent discussions it looks like we're going to call it just "registry" (as it's a convenience layer for fetching remote file system objects, which is not specific to flakes) and it may get constrained to be only available on command line calls.

In any case it would be good to link to documentation on the registry here. I see you just moved that text, but it may be worthwhile improving bits here and there while at it.

Reply:
Good point! I'd be happy to make another PR that replaces all mentions of the flake registry or nix registry with "registry" and link all those occurrences to `nix3-registry.md`.